agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/wasbootstrap/configuration/Configuration.java

```diff
@@ -152,10 +156,16 @@
     }
 
     public static class Heartbeat {
-
         public long intervalSeconds = MINUTES.toSeconds(15);
     }
 
+    public static class Statsbeat {
+        public String ikey = "c4a29126-a7cb-47e5-b348-11414998b11e;IngestionEndpoint"; //workspace-aistatsbeat
```

Comment:
also may as well spell it out completely `instrumentationKey`
```suggestion
        public String ikey = "c4a29126-a7cb-47e5-b348-11414998b11e"; //workspace-aistatsbeat
```